examples/viscosity/dataset.py

```diff
@@ -0,0 +1,45 @@
+import ilthermopy as ilt
+import pandas as pd
+
+import ilthermoml
+
+
+class Dataset(ilthermoml.Dataset):
+    @staticmethod
+    def get_entry_ids() -> list[str]:
+        search = ilt.Search(prop="Viscosity", n_compounds=1)
+
+        def is_binary_smiles(smiles: str) -> bool:
+            return len(smiles.split(".")) == 2  # noqa: PLR2004
+
+        search = search[
+            (search["num_phases"] == 1)
+            & (search["phases"] == "Liquid")
+            & search["cmp1_smiles"].apply(is_binary_smiles)
+        ]
+
+        return list(search["id"])
```

Comment:
```suggestion
        return list(search["id"])[:100]
```

As this is only an example maybe we should only use a smaller sample to speed up fetching.

Reply:
OK - but as soon as you fetch it once, you have everything in cache, so reproducing is a matter of seconds.